src/js/check-ueid.js

```diff
@@ -1,13 +1,30 @@
 import { checkValidity } from './validate';
 import { queryAPI } from './api';
 
+const ENDPOINT = 'https://fac-dev.app.cloud.gov/sac/auditee';
 const FORM = document.forms[0];
 
 function submitForm() {
-  // NOOP for now
-  // const formData = serializeFormData(new FormData(FORM));
+  const formData = serializeFormData(new FormData(FORM));
+  const headers = new Headers();
+
+  headers.append('Content-type', 'application/json');
   /* eslint-disable-next-line no-undef */
-  // queryAPI(ENDPOINT, formData, { authToken, method: 'POST' });
+  headers.append('Authorization', 'Basic ' + authToken); // authToken is set in a script tag right before this script loads
```

Comment:
Since the Login integration is so close to landing, ideally we should start using the real auth solution for hitting the API. Maybe give that till tomorrow to make in into `main` and revise this bit. @timoballard already [got one of the API calls working with the real token](https://github.com/GSA-TTS/FAC-Frontend/pull/24/files#diff-775b4bd55bf763a12fd952e2f5854ab485e9ef93f3a36c79a8f620f62e0f7fd4R13-R29), so you should be able to work off of what he's already done.